### PR TITLE
[Gittigidiyor.net] Update subdomains

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
Dangling reference error pending https://github.com/EFForg/https-everywhere/pull/13773

File: src/chrome/content/rules/Gittigidiyor.xml [hunk 1]
```diff
@@ -18,28 +38,62 @@
...
 
 	<target host="mcdn01.gittigidiyor.net" />
 		<test url="http://mcdn01.gittigidiyor.net/29801/tn24/298011980_tn24_0.jpg" />
 	<target host="mcdnaff03.gittigidiyor.net" />
-	<target host="mhd.gittigidiyor.net" />
 	<target host="st.gittigidiyor.net" />
 	<target host="st2.gittigidiyor.net" />
 
 	<!-- /$ doesn't work with HTTPS -->
-	<exclusion pattern="^http://mhd\.gittigidiyor\.net/$" />
```
Comment (J0WI) [inline src/chrome/content/rules/Gittigidiyor.xml]: Please replace and re-add `mhd.gittigidiyor.net` with `mhd.gittigidiyor.com`

Comment (youdly) [reply src/chrome/content/rules/Gittigidiyor.xml]: Added, resolved conflict and rebased.

**Comment by J0WI** [comment]
Please have a look at Travis and remove the obsolete hosts.

**Comment by youdly** [comment]
Removed preloaded `gittigidiyor.com`

**Comment by J0WI** [comment]
LGTM, thanks!

**Pull request merged**